idealingua-v1/idealingua-v1-transpilers/src/main/scala/izumi/idealingua/translator/toscala/layout/ScalaLayouter.scala

```diff
@@ -122,6 +122,11 @@ class ScalaLayouter(options: ScalaTranslatorOptions) extends TranslationLayouter
 
         val content = keys ++ projDefs ++ Seq(bundle, agg)
 
+        val sbtScalaVersionModule = options.manifest.sbt.scalaVersion.map{
+          v =>
+            Seq(ExtendedModule.RuntimeModule(Module(ModuleId(Seq.empty, "scalaVersion.sbt"), s"scalaVersion in Global := \"$v\"")))
```

Comment:
@Caparow 2.12 build fails, probably because of escaping 
```suggestion
            Seq(ExtendedModule.RuntimeModule(Module(ModuleId(Seq.empty, "scalaVersion.sbt"), s"""scalaVersion in Global := "$v"""")))
```

Reply:
Hmm, thanks!
Missed this one.